test/erlfmt_SUITE_data/simple_comments.erl

```diff
@@ -14,8 +14,9 @@
 -define(STR(X),
     %% comment 1
     ??X
-    %% comment 2
-).
+)
+%% comment 2
+.
```

Comment:
Ah... seeing this now I'm less sure this is the correct way to fix. I assume we'll do this in more cases where there's a "trailing" comment at the end of an attribute, but inside parenthesis. 

I'm not sure we can get away without having some separate space to store the range `.` is covering as opposed to the thing inside parentheses.

Reply:
Good point.  Thanks for helping to fix it :)